<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Graphics::MainWindow</class>
    <widget class="QMainWindow" name="Graphics::MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="palette">
            <palette>
                <active>
                    <colorrole role="Button">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>158</red>
                                <green>158</green>
                                <blue>158</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="Base">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>224</red>
                                <green>224</green>
                                <blue>224</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="Window">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>224</red>
                                <green>224</green>
                                <blue>224</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="PlaceholderText">
                        <brush brushstyle="SolidPattern">
                            <color alpha="128">
                                <red>0</red>
                                <green>0</green>
                                <blue>0</blue>
                            </color>
                        </brush>
                    </colorrole>
                </active>
                <inactive>
                    <colorrole role="Button">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>158</red>
                                <green>158</green>
                                <blue>158</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="Base">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>224</red>
                                <green>224</green>
                                <blue>224</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="Window">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>224</red>
                                <green>224</green>
                                <blue>224</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="PlaceholderText">
                        <brush brushstyle="SolidPattern">
                            <color alpha="128">
                                <red>0</red>
                                <green>0</green>
                                <blue>0</blue>
                            </color>
                        </brush>
                    </colorrole>
                </inactive>
                <disabled>
                    <colorrole role="Button">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>158</red>
                                <green>158</green>
                                <blue>158</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="Base">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>224</red>
                                <green>224</green>
                                <blue>224</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="Window">
                        <brush brushstyle="SolidPattern">
                            <color alpha="255">
                                <red>224</red>
                                <green>224</green>
                                <blue>224</blue>
                            </color>
                        </brush>
                    </colorrole>
                    <colorrole role="PlaceholderText">
                        <brush brushstyle="SolidPattern">
                            <color alpha="128">
                                <red>0</red>
                                <green>0</green>
                                <blue>0</blue>
                            </color>
                        </brush>
                    </colorrole>
                </disabled>
            </palette>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <property name="autoFillBackground">
            <bool>false</bool>
        </property>
        <property name="styleSheet">
            <string notr="true"/>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QWidget" name="controlsLayout" native="true">
                        <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1">
                            <item row="0" column="2" alignment="Qt::AlignRight">
                                <widget class="QSpinBox" name="bpmSpin">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1" alignment="Qt::AlignHCenter">
                                <widget class="QWidget" name="horizontalWidget" native="true">
                                    <layout class="QHBoxLayout" name="playStopLayout">
                                        <item>
                                            <widget class="QPushButton" name="playButton">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="styleSheet">
                                                    <string notr="true">height: 35px; width: 35px;</string>
                                                </property>
                                                <property name="text">
                                                    <string/>
                                                </property>
                                                <property name="icon">
                                                    <iconset resource="Resources/Icons/icons.qrc">
                                                        <normaloff>:/icons/play_button.png</normaloff>
                                                        :/icons/play_button.png
                                                    </iconset>
                                                </property>
                                                <property name="checkable">
                                                    <bool>true</bool>
                                                </property>
                                                <property name="checked">
                                                    <bool>false</bool>
                                                </property>
                                                <property name="flat">
                                                    <bool>false</bool>
                                                </property>
                                            </widget>
                                        </item>
                                        <item>
                                            <widget class="QPushButton" name="stopButton">
                                                <property name="sizePolicy">
                                                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                        <horstretch>0</horstretch>
                                                        <verstretch>0</verstretch>
                                                    </sizepolicy>
                                                </property>
                                                <property name="styleSheet">
                                                    <string notr="true">height: 35px; width: 35px;</string>
                                                </property>
                                                <property name="text">
                                                    <string/>
                                                </property>
                                                <property name="icon">
                                                    <iconset resource="Resources/Icons/icons.qrc">
                                                        <normaloff>:/icons/stop_button.png</normaloff>
                                                        :/icons/stop_button.png
                                                    </iconset>
                                                </property>
                                            </widget>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item row="0" column="0">
                                <spacer name="horizontalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                        <property name="sizeConstraint">
                            <enum>QLayout::SetDefaultConstraint</enum>
                        </property>
                        <item>
                            <spacer name="horizontalSpacer_3">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="zoomPlusButton">
                                <property name="text">
                                    <string/>
                                </property>
                                <property name="icon">
                                    <iconset resource="Resources/Icons/icons.qrc">
                                        <normaloff>:/icons/plus_button.png</normaloff>:/icons/plus_button.png
                                    </iconset>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="zoomLessButton">
                                <property name="text">
                                    <string/>
                                </property>
                                <property name="icon">
                                    <iconset resource="Resources/Icons/icons.qrc">
                                        <normaloff>:/icons/less_button.png</normaloff>:/icons/less_button.png
                                    </iconset>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>24</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>File</string>
                </property>
                <addaction name="actionLoad_project"/>
                <addaction name="actionSave_project"/>
                <addaction name="separator"/>
                <addaction name="actionImport_file"/>
            </widget>
            <widget class="QMenu" name="menuEdit">
                <property name="title">
                    <string>Edit</string>
                </property>
                <addaction name="actionUndo"/>
                <addaction name="actionRedo"/>
                <addaction name="separator"/>
                <addaction name="actionOptions"/>
            </widget>
            <widget class="QMenu" name="menuHelp">
                <property name="title">
                    <string>Help</string>
                </property>
                <addaction name="actionAbout"/>
            </widget>
            <addaction name="menuFile"/>
            <addaction name="menuEdit"/>
            <addaction name="menuHelp"/>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <action name="actionAbout">
            <property name="text">
                <string>About</string>
            </property>
        </action>
        <action name="actionLoad_project">
            <property name="text">
                <string>Load project</string>
            </property>
        </action>
        <action name="actionSave_project">
            <property name="text">
                <string>Save project</string>
            </property>
        </action>
        <action name="actionImport_file">
            <property name="text">
                <string>Import file</string>
            </property>
        </action>
        <action name="actionUndo">
            <property name="text">
                <string>Undo</string>
            </property>
        </action>
        <action name="actionRedo">
            <property name="text">
                <string>Redo</string>
            </property>
        </action>
        <action name="actionOptions">
            <property name="text">
                <string>Options</string>
            </property>
        </action>
    </widget>
    <resources>
        <include location="Resources/Icons/icons.qrc"/>
    </resources>
    <connections/>
</ui>
